tailwind.config.js

```diff
@@ -1,5 +1,4 @@
-const { screens } = require('./breakpoints.js');
-
+/* eslint-disable vue/sort-keys */
```

Comment:
Why did we remove `breakpoints.js` here?

Reply:
I didn't see it being used anywhere else, and logically if someone needs these values they can be accessed from that config file if needed.